improver_tests/acceptance/test_convert_to_realizations.py

```diff
@@ -58,6 +58,24 @@ def test_percentiles(tmp_path):
     acc.compare(output_path, kgo_path)
 
 
+@pytest.mark.slow
+def test_percentiles_reordering(tmp_path):
+    """Test percentile to realization conversion with reordering"""
+    kgo_dir = acc.kgo_root() / \
+        "percentiles-to-realizations/percentiles_reordering"
```

Comment:
Please copy these input and output acceptance test data into a new directory for this CLI (and repeat for all data used by this file) - this will make it easier for us to retire the percentiles-to-realizations CLI as we are planning.

Reply:
I've now copied the `convert-to_realizations` CLI inputs and outputs into a separate directory.